rust/nft-candy-machine/src/lib.rs

```diff
@@ -26,7 +26,21 @@ pub mod nft_candy_machine {
     };
 
     use super::*;
+    pub fn withdraw_funds<'info>(
+        ctx: Context<'_, '_, '_, 'info, WithdrawFunds<'info>>,
+    ) -> ProgramResult {
+        let authority = &ctx.remaining_accounts[0];
```

Comment:
Why are you using remaining accounts instead of ctx.accounts.authority and ctx.accounts.config? This is not good hygiene.

Reply:
Trial and error til something works haha